widgetssdk/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'kotlin-parcelize'
 apply plugin: 'app.cash.paparazzi'
 
 android {
-  compileSdkVersion 33
+  compileSdkVersion 34
```

Comment:
We agreed to upgrade the compile SDK version to 33, so integrators aren't forced to upgrade to the latest version. Upgrading this to the latest available API version will force the integrators to upgrade their dependencies, especially `core-ktx` to the latest version. I'd prefer to keep the testing app compile and target SDK versions latest API version(34), and Widget and core sdk target and compile sdk versions one version lowe(33).

Reply:
Yes, indeed. I was about to write this also.